generator/generator.yml

```
@@ -1144,3 +1144,26 @@ modules:
       yrfUpTime:
         help: The time (in seconds) since the network management portion of the system was last re-initialized - 1.3.6.1.4.1.1182.2.2.4
         scale: 0.01
+
+# MSA fieldserver gateways
+#
+# Gateway configuration guides:
+# * General info: https://cdn.chipkin.com/assets/uploads/2022/Mar/FieldServer_Configuration_Guide_03-19-58-11.pdf
+# * SNMP: see "Protocol Driver Manual - SNMP STD" at https://us.msasafety.com/smc/protocols
+  fieldserver:
+    walk:
+      - analogInputsTable
```

Would you mind making these fully qualified?

```suggestion
      - "FIELDSERVER-STD-MIB-v1-00aF::analogInputsTable"
```

---

Of course not. Done.